package.json

```diff
@@ -1,14 +1,16 @@
 {
-  "name": "spring-nft-contracts",
-  "version": "0.0.1",
-  "description": "WeTrust Spring Smart Contracts",
-  "main": "truffle-config.js",
+  "name": "@wetrustplatform/spring-nft-contracts",
+  "version": "1.0.0",
```

Comment:
@yuantu1919 let's keep this 0.0.1

There's no change to the implementation of nft smart contract.

Reply:
ok